docs/intro/overview.rst

```diff
@@ -19,74 +19,69 @@ Walk-through of an example spider
 In order to show you what Scrapy brings to the table, we'll walk you through an
 example of a Scrapy Spider using the simplest way to run a spider.
 
-So, here's the code for a spider that follows the links to the top
-voted questions on StackOverflow and scrapes some data from each page::
+Here's the code for a spider that scrapes famous quotes from website
+http://quotes.toscrape.com, following the pagination::
 
     import scrapy
 
 
-    class StackOverflowSpider(scrapy.Spider):
-        name = 'stackoverflow'
-        start_urls = ['http://stackoverflow.com/questions?sort=votes']
+    class QuotesSpider(scrapy.Spider):
+        name = "quotes"
+        start_urls = [
+            'http://quotes.toscrape.com/tag/humor/',
+        ]
 
         def parse(self, response):
-            for href in response.css('.question-summary h3 a::attr(href)'):
-                full_url = response.urljoin(href.extract())
-                yield scrapy.Request(full_url, callback=self.parse_question)
+            for quote in response.css('div.quote'):
+                yield {
+                    'text': quote.css('span.text::text').extract_first(),
+                    'author': quote.xpath('span/small/text()').extract_first(),
+                }
 
-        def parse_question(self, response):
-            yield {
-                'title': response.css('h1 a::text').extract_first(),
-                'votes': response.css('.question .vote-count-post::text').extract_first(),
-                'body': response.css('.question .post-text').extract_first(),
-                'tags': response.css('.question .post-tag::text').extract(),
-                'link': response.url,
-            }
+            next_page = response.css('li.next a::attr("href")').extract_first()
+            if next_page:
```

Comment:
I think `next_page is None` could be slightly better because an empty string is a valid relative URL